### PR TITLE
feat: added images component

### PR DESCRIPTION
**Pull request by developer-anand**

### CODE REVIEW

File: components/images/image.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  border: 1px solid var(--black);
+  align-items: center;
+  justify-content: center;
+}
+
+.cir-img {
+  width: 100%;
+  height: auto;
+  max-width: 300px;
+  border-radius: 50%;
```
Comment (srishtigautam21) [inline components/images/image.css:30]: use rem for more than 10px

File: components/images/image.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  object-fit: cover;
+}
+
+.res-img {
+  padding: auto;
+  margin: 2rem;
+  width: 100%;
+  height: auto;
+  max-width: 800px;
+}
```
Comment (srishtigautam21) [inline components/images/image.css:70]: use rem

**Review by srishtigautam21 — COMMENTED** [review]
change px to rem where ever px is greater than 10

File: components/images/image.css (hunk 1 above)
Comment (developer-anand) [reply components/images/image.css:30]: done

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Review by srishtigautam21 — APPROVED** [review]
good to go

**Pull request merged**